src/runner/at-driver.js

```diff
@@ -128,6 +141,18 @@ export class ATDriver {
       }
     }
   }
+
+  /**
+   *
+   * @param {[{name: string, value: string | boolean}]} settings
+   */
+  async setSettings(settings) {
+    if (settings?.length) {
+      const { atName } = await this.getCapabilities();
+      const method = atName == 'nvda' ? 'nvda:settings.setSettings' : 'settings.setSettings';
```

Comment:
I'm not thrilled about papering over nonconformance (what are we writing jQuery over here?), but you're probably right to do this.

Reply:
Awesome! Let's track it!